geode-connectors/src/distributedTest/java/org/apache/geode/connectors/jdbc/internal/cli/CreateMappingCommandDUnitTest.java

```diff
@@ -114,20 +113,33 @@ public void createExistingRegionMappingFails() {
     CommandStringBuilder csb = new CommandStringBuilder(CREATE_MAPPING);
     csb.addOption(CREATE_MAPPING__REGION_NAME, REGION_NAME);
     csb.addOption(CREATE_MAPPING__DATA_SOURCE_NAME, "connection");
+    csb.addOption(CREATE_MAPPING__PDX_NAME, "myPdxClass");
     csb.addOption(CREATE_MAPPING__TABLE_NAME, "myTable");
     gfsh.executeAndAssertThat(csb.toString()).statusIsSuccess();
 
     csb = new CommandStringBuilder(CREATE_MAPPING);
     csb.addOption(CREATE_MAPPING__REGION_NAME, REGION_NAME);
     csb.addOption(CREATE_MAPPING__DATA_SOURCE_NAME, "connection");
+    csb.addOption(CREATE_MAPPING__PDX_NAME, "myPdxClass");
     csb.addOption(CREATE_MAPPING__TABLE_NAME, "bogus");
     gfsh.executeAndAssertThat(csb.toString()).statusIsError();
 
     locator.invoke(() -> {
       String xml = InternalLocator.getLocator().getConfigurationPersistenceService()
           .getConfiguration("cluster").getCacheXmlContent();
       assertThat(xml).isNotNull().contains("jdbc:mapping").contains("myTable")
+          .contains("myPdxClass")
           .doesNotContain("bogus");
     });
   }
+
+  @Test
+  public void createMappingWithoutPdxNameFails() {
+    CommandStringBuilder csb = new CommandStringBuilder(CREATE_MAPPING);
+    csb.addOption(CREATE_MAPPING__REGION_NAME, REGION_NAME);
+    csb.addOption(CREATE_MAPPING__DATA_SOURCE_NAME, "connection");
+    // NOTE: --table is optional so it should not be in the ouput but it is. See GEODE-3468.
+    gfsh.executeAndAssertThat(csb.toString()).statusIsError()
+        .containsOutput("You should specify option (--table, --pdx-name) for this command");
```

Comment:
About error message, "You should" not sure if this the pattern we use across the error messages. 
How about -  "missing x y" see command help for "create mapping"

Reply:
For improvements to this message, make your comments on GEODE-3468.
I agree with you that this message should be improved.